static/js/containers/pages/DashboardPage.js

```diff
@@ -193,13 +200,22 @@ export class DashboardPage extends React.Component<
 
   renderEmailSettingsDialog(enrollment: RunEnrollment) {
     const { emailSettingsModalVisibility } = this.state
+    let isOpen = false
+    if (emailSettingsModalVisibility[enrollment.id] === undefined) {
+      isOpen = false
+    } else {
+      isOpen = emailSettingsModalVisibility[enrollment.id]
+    }
```

Comment:
This is the other place as mentioned in my above comment.

Reply:
here we are not updating anything just fetching it out